dev_docs/developer_guidelines.md

```diff
@@ -21,20 +53,24 @@
 If you install cellxgene using `make install-dev` the server will be restarted every time you make changes on the server code. If changes affects the client, the browser must be reloaded.
 
 ### Linter
-We use `flake8` to lint code. Travis CI runs `flake8 server`.
+
+We use `yapf` to auto-format and lint python.
```

Comment:
s/yapf/black/g

Reply:
also, unless I'm misreading the Makefile, we use black to auto-format, and flake8 to lint.

Reply:
Good catch 👍 Fixed this.